test/controllers/TransferSummaryControllerSpec.scala

```diff
@@ -155,6 +155,7 @@ class TransferSummaryControllerSpec extends FrontEndTestHelper {
     "redirects to the transfer confirmation page when a valid form is submitted" in {
```

Comment:
Should this read: "... to the transfer **complete** page .."?

Reply:
I've changed this one as well.